hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,590 @@
+                    BYTE_A, and BYTE_B.
+                    ''',
+          resval: "0x0"
+        },
+        { bits:  "15:8",
+          name:  "BYTE_B",
+          desc:  '''One of four bytes to transmit.  If ByteOrder = 0,
+                    this byte will be transmitted third after BYTE_D and BYTE_C.
+                    Otherwise BYTE_B will be transmitted second, after BYTE_A.
+                    ''',
+          resval: "0x0"
+        },
+        { bits:  "7:0",
+          name:  "BYTE_A",
+          desc:  '''One of four bytes to transmit.  If ByteOrder = 0,
+                    This byte will be transmitted last, after BYTE_D, BYTE_C
+                    and BYTE_B.  Otherwise BYTE_A will be transmitted first.
+                    ''',
+          resval: "0x0"
+        }
+      ]
+    },
+    { name: "RXDATA",
+      desc: "SPI Receive data",
+      swaccess: "ro",
+      hwaccess: "hwo",
+      fields: [
+        { bits:  "31:24",
+          name:  "BYTE_D",
+          desc:  '''One of four bytes to receive.  If ByteOrder = 0,
+                    this byte will be received before BYTE_C, BYTE_B
+                    and BYTE_A. Otherwise BYTE_D will be received
+                    last of the four.
+                    ''',
+          resval: "0x0"
+        },
+        { bits:  "23:16",
+          name:  "BYTE_C",
+          desc:  '''One of four bytes to receive.  If ByteOrder = 0,
+                    this byte will be received second, after BYTE_D.
+                    Otherwise BYTE_C will be received third, after
+                    BYTE_A, and BYTE_B.
+                    ''',
+          resval: "0x0"
+        },
+        { bits:  "15:8",
+          name:  "BYTE_B",
+          desc:  '''One of four bytes to receive.  If ByteOrder = 0,
+                    this byte will be received third after BYTE_D
+                    and BYTE_C.  Otherwise BYTE_B will be received
+                    second, after BYTE_A.
+                    ''',
+          resval: "0x0"
+        },
+        { bits:  "7:0",
+          name:  "BYTE_A",
+          desc:  '''One of four bytes to receive.  If ByteOrder = 0,
+                    This byte will be received last, after BYTE_D, BYTE_C
+                    and BYTE_B.  Otherwise BYTE_A will be received first.
+                    ''',
+          resval: "0x0"
+        }
+      ]
+    },
+    { name: "ERROR_ENABLE",
+      desc: "Controls which classes of errors raise an interrupt."
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "CMDERR",
+          desc: '''Command Error: Assert to send an error interrupt whenever
+                   a command is issued while BUSY is asserted'''
+          resval: "0x1"
+        },
+        { bits: "1",
+          name: "OVERFLOW",
+          desc: '''Overflow Errors: Assert to send an error interrupt whenever
+                   the TXFIFO overflows.'''
```

Comment:
I'm not keen on the TXFIFO name. We've got a capitalized register name, "!!TXDATA", but I'd suggest something with lower case letters and/or spaces for stuff that's not a register. So maybe this should be something like "whenever the TX FIFO overflows".

Reply:
Sure if you prefer TX FIFO.  No problem.  Lol, but I do wish I'd seen this comment before converging them all from TX FIFO to TXFIFO However search and replace is easy enough.  ;)

Reply:
All instances of "TXFIFO" in text changed to "TX FIFO"